constructorio-client/src/main/java/io/constructor/client/models/FinalizeQuizResponse.java

```diff
@@ -0,0 +1,27 @@
+package io.constructor.client.models;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Constructor.io Finalize Quiz Response ... uses Gson/Reflection to load data in
+ */
+public class FinalizeQuizResponse {
```

Comment:
Is it possible or would it make sense to consolidate `FinalizeQuizResponse` and `NextQuizResponse` into a single `QuizResponse` object?